test/controllers/organizations/inboxes_controller_test.rb

```diff
@@ -32,12 +32,16 @@ class InboxesControllerTest < ActionDispatch::IntegrationTest
   end
 
   test "should create inbox" do
+    ActsAsTenant.current_tenant = @organization
```

Comment:
This line looks like a perfect candidate for some  kind of `before(:all) { ActsAsTenant.current_tenant = @organization }` block.

Reply:
acts_as_tenant [does not work well](https://jumpstartrails.com/discussions/testing-with-config-require_tenant-true) with `assert_difference`. I did not find a better way than this

Reply:
can we add RSpec?

Reply:
I want to stick to the defaults as much as possible.

If someone wants to replace minitest with rspec in their project, they are free to do so. It's really easy with AI

Reply:
oh... OK